dotnet/src/SemanticKernel.Core/SemanticKernel.Core.csproj

```diff
@@ -26,6 +26,11 @@
     <InternalsVisibleTo Include="SemanticKernel.Extensions.UnitTests" />
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection" />
```

Comment:
Do you have to reference `Microsoft.Extensions.DependencyInjection`? It would be best if you could just reference the abstractions.

Reply:
Yes, that's where BuildServiceProvider lives.